src/domain/provider-service.ts

```diff
@@ -88,4 +106,17 @@ export class ProviderService {
     this.statusPublisher.publish(status)
     this.lastStatus = status
   }
+
+  private serviceStatusDTOToModel (status: ServiceStatusDTO): ServiceStatus {
+    if (status === ServiceStatusDTO.NOT_RUNNING) {
+      return ServiceStatus.NOT_RUNNING
+    }
+    if (status === ServiceStatusDTO.STARTING) {
+      return ServiceStatus.STARTING
+    }
+    if (status === ServiceStatusDTO.RUNNING) {
+      return ServiceStatus.RUNNING
+    }
+    throw new Error(`Unknown status: ${status}, ${ServiceStatusDTO.NOT_RUNNING}`)
```

Comment:
what purpose of `${ServiceStatusDTO.NOT_RUNNING}` here ?

Reply:
Good catch - it was just a leftover from debugging :D